lib/mbta.js

```diff
@@ -4,7 +4,30 @@
 
 const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
 
-};
+  let Subway = {
+  "Red Line": ["South Station", "Park Street", "Kendall", "Central", "Haymarket", "Porter", "Davis", "Alewife"],
+  "Green Line": ["Government Center", "Park Street", "Boylston", "Arlington", "Copley", "Hynes", "Kenmore"],
+  "Orange Line": ["North Station", "Haymarket", "Park Street", "State", "Downtown Crossing", "Chinatown", "Back Bay", "Forest Hills"]
+
+}
+
+if (startLine = endLine){
+  var a = Subway[startLine].indexOf(startStation);
+  var b = Subway[startLine].indexOf(endStation);
+  var c = abs(a-b);
+  return c;
+}
+
+else{
```

Comment:
when writing an if/else you want to put you else right after the close if bracket so:
```
return c;
 } else {
```